script.js

```diff
@@ -1,5 +1,202 @@
+    Type 2 and click enter if you wish for dice 2 to be the first number`;
+    return myOutputValue
+  }
+  else if (choice == 1){
+    myOutputValue = `${playerNumber} have chose dice 1 to be the first in order, your result is ${playerArray[0]}${playerArray[1]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[0]}${playerArray[1]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+  else if (choice == 2){
+    myOutputValue = `${playerNumber} have chose dice 2 to be the first in order, your result is ${playerArray[1]}${playerArray[0]} <br><br>
+    Click enter for next.`;
+
+    console.log(`player ${playerNumber} dice results: ${playerArray[1]}${playerArray[0]}`);
+    console.log(`user dice order: ${choice}`);
+    return myOutputValue
+  }
+}
+
+// 3 conditions: 
+// a > b
+// b < a
+// a == b
+var gameOutcome = function () {
+  if (Number(player1diceResults.join("")) > Number(player2diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player1} for winning!!! <br><br>
+    ${player1} BEATS ${player2} with a score of ${Number(player1diceResults.join(""))} against ${Number(player2diceResults.join(""))} <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  } 
+  else if (Number(player2diceResults.join("")) > Number(player1diceResults.join(""))) {
+    myOutputValue = `Congratulations ${player2} for winning!!! <br><br>
+     ${player2} BEATS ${player1} with a score of ${Number(player2diceResults.join(""))} against ${Number(player1diceResults.join(""))}<br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+  else if (Number(player2diceResults.join("")) == Number(player1diceResults.join(""))){
+    myOutputValue = `Issa Draw lubba dubba dub dub <br><br>
+    Click enter to start another new game`;
+    return myOutputValue;
+  }
+};
+
+
+
+
+currentGameMode = "Welcome to Beat That"
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  console.log("test");
-  return myOutputValue;
+  if (currentGameMode == "Welcome to Beat That"){
+    myOutputValue = `Welcome to BEAT THAT!<br><br>
+     Player 1 please enter your name and click ENTER, followed by player 2 name and click ENTER to start the game. <br><br>
+      Game Rules: <br>
+      1. This is a 2 person, player vs player game. <br>
+      2. The game will roll 2 dice and show the dice rolls result. For example, dice 1 rolled 3 and dice 2 rolled 4. <br>
+      3. The player that rolled the dice will pick the order of the dice.(If they wanted the number 43, they would specify that the 2nd dice goes first.) <br>
+      4. After both players have rolled and chosen their dice order, the player with the higher combined number wins.  `;
+
+    currentGameMode = "Enter Player 1 name";
+    console.log(`currentGameMode: ${currentGameMode}`);
+
+    return myOutputValue;
+  }
```

Comment:
this is ok, but try seeing how if you can output the message into the output box immediately on page load by editing the html so that the user sees the message straight away when he loads the game. Currently when I load your game, I am greeted with an empty text input and output box, and it is not immediately clear what I have to do. Always think about user experience when you are designing something!

This will involve editing the html which I realise you didn't have a chance to in this project but do try playing around with HTML/CSS for blackjack! Especially so since you're interested in bootcamp. The first part of bootcamp is mainly front-end doing DOM manipulation using HTML/CSS and JavaScript so it would be good to have some exposure right now, even if it is simple things like changing the background color of the page or the container box, or changing the fonts that are being used.